packages/plugin-core/src/components/pods/PodControls.ts

```diff
@@ -288,4 +306,46 @@ export class PodUIControls {
     qp.items = items;
     return qp;
   }
+
+  /**
+   * Create a new configuration file for a service connection
+   * @param serviceType
+   * @param id a unique ID to identify this connection
+   * @returns full path to the newly created config file
+   */
+  public static async createNewConfig({
```

Comment:
Why'd this function get moved to PodControls?

Reply:
Had to discuss regarding this function. To migrate gdoc ouath flow in `ExternalConnectionManager` from `plugin-core/pods.ts`  we require google oauth client id which is in plugin core. I wasn't sure about the best way to provide client id to connection manager when the external service is google.

Reply:
I don't think we need to make the client ID a part of the serialized config. The Client ID should be reused across all gdoc connections.